SendITapp/api/v1/views/parcelview.py

```diff
@@ -16,55 +16,46 @@ def get(self):
-
-            if key == 'destination' or key =='recipient':
-                if data['destination'] == "" or data['recipient'] == ""\
-                or data['recipient'].isalpha()== False or data['destination'].isalpha()==False:
-                    message = 'Check your details. Destination, Recipient...'
-                    payload = {"Status": "Failed", "Message": message}
-                    answ = make_response(jsonify(payload), 400)
-                    answ.content_type = 'application/json;charset=utf-8'
-                    return answ
-
-            if key == 'price' or key == 'weight':
-                if data['price'] == "" or data['weight'] == "":
-                    message = 'Kindly confirm the wight and pricing of your package'
-                    payload = {"Status": "Failed", "Message": message}
-                    answ = make_response(jsonify(payload), 400)
-                    answ.content_type = 'application/json;charset=utf-8'
-                    return answ
-            if key == 'status':
-                accepted=["Pending","Canceled","Delivered"]
-                if data['status'] not in accepted:
-                        message = 'Status can only be Pending, Delivered or Canceled'
-                        payload = {"Status": "Failed", "Message": message, "Data": data['status']}
-                        answ = make_response(jsonify(payload), 400)
-                        answ.content_type = 'application/json;charset=utf-8'
-                        return answ
+        """Check email validation."""            
+        if data['sender'] == "" or data['sender'].isalpha()==False:
+            message = 'Please confirm the sender'
+            payload = {"Status": "Failed", "Message": message}
+            answ = make_response(jsonify(payload), 400)
+            return answ
+
+    
+        if data['destination'] == "" or data['recipient'] == ""\
+        or data['recipient'].isalpha()== False or data['destination'].isalpha()==False:
+            message = 'Check your details. Destination, Recipient...'
+            payload = {"Status": "Failed", "Message": message}
+            answ = make_response(jsonify(payload), 400)
+            return answ
+
+    
+        if data['price'] == "" or data['weight'] == "":
+            message = 'Kindly confirm the wight and pricing of your package'
+            payload = {"Status": "Failed", "Message": message}
+            answ = make_response(jsonify(payload), 400)
+            return answ
+    
+        accepted=["Pending","Canceled","Delivered"]
+        if data['status'] not in accepted:
+                message = 'Status can only be Pending, Delivered or Canceled'
+                payload = {"Status": "Failed", "Message": message, "Data": data['status']}
+                answ = make_response(jsonify(payload), 400)
+                return answ
 
         par1 = Parcel()
         par1.create_parcel(data["destination"], data["recipient"], data["sender"],
```

Comment:
Let this method return the created parcel. You should not return what is in `data`